website/static/js/comment.js

```diff
@@ -77,19 +261,50 @@ var BaseComment = function() {
 
     self.replying = ko.observable(false);
     self.replyContent = ko.observable('');
+    self.replyMentions = ko.observableArray([]);
+
+    self.saveContent = ko.computed(function() {
+        var content = self.replyContent();
+        if (!content) {
+            content = '';
+        }
```

Comment:
As with https://github.com/leb2dg/osf.io/pull/14#discussion_r61760728, we can fold this down to:

```
var content = self.replyContent() || '';
```